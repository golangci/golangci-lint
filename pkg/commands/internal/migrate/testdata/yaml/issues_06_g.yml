issues:
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

  exclude-use-default: true
  # Related to common-false-positives
  include:
    - EXC0006
    - EXC0007
