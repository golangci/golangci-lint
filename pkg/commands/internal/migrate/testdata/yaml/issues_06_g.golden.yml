version: "2"
linters:
  exclusions:
    presets:
      - comments
      - legacy
      - std-error-handling
